tests/acceptance/features/apiSpacesShares/moveSpaces.feature

```diff
@@ -59,13 +59,13 @@ Feature: move (rename) file
       | shareWith | Alice       |
       | role      | <from_role> |
     When user "Alice" moves file "project1.txt" from space "Project1" to "project1.txt" inside space "Project2" using the WebDAV API
-    Then the HTTP status code should be "<https_status_code>"
+    Then the HTTP status code should be "<http-status-code>"
     And for user "Alice" the space "Project1" should contain these entries:
       | project1.txt |
     And for user "Alice" the space "Project2" should not contain these entries:
       | project1.txt |
     Examples:
-      | from_role | to_role | https_status_code |
+      | from_role | to_role | http-status-code  |
       | manager   | manager | 502               |
       | editor    | manager | 502               |
       | manager   | editor  | 502               |
```

Comment:
according https://github.com/owncloud/ocis/issues/8116 issue we expect here 403?

Reply:
yeah, it should be 403? Should I expect 403 and put them in expected failure list?

Reply:
I think yes

Reply:
okey thanks!